bower.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "purescript-node-net",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/purescript-node/purescript-node-net"
+  },
+  "license": "MIT",
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "output"
+  ],
+  "dependencies": {
+    "purescript-effect": "1.0.0 - 2",
+    "purescript-either": "1.0.0 - 4",
+    "purescript-exceptions": "4.0.0 - 4",
+    "purescript-foreign": "5.0.0 - 5",
+    "purescript-maybe": "1.0.0 - 4",
+    "purescript-node-buffer": "5.0.0 - 5",
+    "purescript-node-fs": "5.0.0 - 5",
+    "purescript-nullable": "3.0.0 - 4",
+    "purescript-options": "1.0.0 - 5",
+    "purescript-prelude": "4.0.0 - 4",
+    "purescript-transformers": "1.0.0 - 4"
+  },
+  "devDependencies": {
+    "purescript-console": "4.0.0 - 4"
```

Comment:
This seems a little bit strange to me. Does this mean the same as `>= 4.0.0 < 5`? If so, I think I'd prefer that; it seems clearer.

Reply:
Using just `4` is an option too, as that also means `>= 4.0.0 < 5.0.0` according to https://jubianchi.github.io/semver-check/#/constraint/4

Reply:
That also seems less obvious, and less workable if you want to say something like `>= 4.1.1 < 5.0.0`. I think using the `>= x.y.z < a.b.c` scheme everywhere is ideal, because it can be used for any range we might want to express, and because you'll never need to look it up to find out what it means.

Reply:
That would be clearer, but I can't seem to get that syntax to work for me.

This is the error I get:
```
yarn run bower install
yarn run v1.13.0
$ /home/joneshf/programming/purescript-node/purescript-node-net/node_modules/.bin/bower install
bower purescript-console#>= 4.0.0 && < 5       not-cached https://github.com/purescript/purescript-console.git#>= 4.0.0 && < 5
bower purescript-console#>= 4.0.0 && < 5          resolve https://github.com/purescript/purescript-console.git#>= 4.0.0 && < 5
bower purescript-console#>= 4.0.0 && < 5     ENORESTARGET Tag/branch >= 4.0.0 && < 5 does not exist

Additional error details:
Available tags: v0.1.0, v0.1.0-rc.1, v0.1.1, v1.0.0, v1.0.0-rc.1, v2.0.0, v3.0.0, v4.0.0, v4.1.0, v4.2.0
Available branches: master
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```
Maybe I'm doing something wrong. If you can make it work, I'm more than happy to use it.

Reply:
Try removing the `&&`?

Reply:
Oh wow. That was it.